### ISSUE TITLE
Incorrect warning concat, map (from different modules)

### ISSUE THREAD

**Issue by retep007** [ndmitchell/hlint#385]
concat is from Data.Text, map is from Prelude
```
Use concatMap
|| Found:
||   concat $ map buildMessage inc
|| Why not:
||   concatMap buildMessage inc
```
